docker/start.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
```

Comment:
Was this script copied from somewhere? If so, could we link the original? if not, could we add comments on what exactly we are doing on every line as it looks very arbitrary and hard to digest?